### PR TITLE
New procrustes

### PR DESCRIPTION
**Pull request by wasade**
Tests and an action to drive the method

@gibsramen, how's this look? 

We could compute disparity for the paired samples, but it isn't directly usable in emperor as its procrustes action requires two ordinations for input. In this case, we merge the ordination space into a single object (which I think is what we want to do...)

### CODE REVIEW

File: q2_diversity/plugin_setup.py [hunk 1]
```diff
@@ -357,6 +357,39 @@
...
+    ],
+    input_descriptions={
+        'reference': ('The ordination matrix to which data is fitted to.'),
+        'other': ("The ordination matrix that's fitted to the reference "
+                  "ordination."),
+    },
+    parameter_descriptions={
+        'dimensions': ('The number of dimensions to use when fitting the two '
+                       'matrices'),
+        'pairing': ("The metadata column describing the pair a sample may "
```
Comment (gibsramen) [inline q2_diversity/plugin_setup.py]: Description of `pairing` parameter needs a fix.

**Review by gibsramen — CHANGES_REQUESTED** [review]
Thanks! Minor documentation issue but aside from that I think this is great.

File: q2_diversity/plugin_setup.py (hunk 1 above)
Comment (wasade) [reply q2_diversity/plugin_setup.py]: Done!

**Comment by wasade** [comment]
Hey @gibsramen, I think this is good now

**Comment by gibsramen** [comment]
Agreed, thanks!

**Pull request merged**